website/src/styles/globals.css

```diff
@@ -0,0 +1,54 @@
+@tailwind base;
+@tailwind components;
+@tailwind utilities;
+
+/* html,
+body {
+  padding: 0;
```

Comment:
To be reactivated later or could these styles be removed?

Reply:
removed